packages/eslint-config/package.json

```diff
@@ -33,11 +33,13 @@
         "npm-run-all": "^4.1.5"
     },
     "devDependencies": {
+        "@comet/eslint-plugin": "workspace:^7.7.0",
         "eslint": "^8.32.0",
         "prettier": "^2.0.0",
         "typescript": "^4.0.0"
     },
     "peerDependencies": {
+        "@comet/eslint-plugin": "workspace:^7.7.0",
```

Comment:
It's already a normal dependency. Why add it as a peer dependency? This would be a breaking change.

Reply:
if the user of the eslint-config does not install this as an dependency - eslint is not working correctly. I thought it would be a good idea to add it to the peer Dependencies, so user of the library, at least know that they have to install it.

Reply:
But the plugin is a dependency of the config 🤔